### ISSUE TITLE
Debian support

### ISSUE THREAD

**Issue by MrCoolAndroid** [SpacingBat3/nativefier-discord-arm#1]
Hey! Thanks for creating this, I really wanted Discord for my ARM device, sadly there's no official version, hope u can build this for Debian based distros, if there is something I can help with tell me! I have an ARM device If it's needed for test

**Comment by SpacingBat3** [comment]
@MrCoolAndroid, I've made [devel](https://github.com/SpacingBat3/nativefier-discord-arm/tree/devel) branch that has commit [c78a957](https://github.com/SpacingBat3/nativefier-discord-arm/commit/c78a9572a6ac2660d0dd9780aa350bfc71d35d46), which partialy allows you to install this repo on Debian (you can't still install it as DEB package for `apt`/`apt-get`/`dpkg`). Also be aware that it won't provide binaries, so you need to provide nativefier on your distro by yourself.

**Comment by MrCoolAndroid** [comment]
> @MrCoolAndroid, I've made [devel](https://github.com/SpacingBat3/nativefier-discord-arm/tree/devel) branch that has commit [c78a957](https://github.com/SpacingBat3/nativefier-discord-arm/commit/c78a9572a6ac2660d0dd9780aa350bfc71d35d46), which partialy allows you to install this repo on Debian (you can't still install it as DEB package for `apt`/`apt-get`/`dpkg`). Also be aware that it won't provide binaries, so you need to provide nativefier on your distro by yourself.

Ok! I will try it out

**Comment by SpacingBat3** [comment]
@MrCoolAndroid , I've checked how things are on Debian...
There's no equaliment for PKGBUILD on Debian.

I could make one with `bash` script, but that would take me quite a long time... And this would be better to make as another project, unassociated with Discord at all.

I would need sources (which **are not** produced by nativefier) containing Makefile, so I could use then `dh_make`. I haven't find yet any other way to create script on Debian similar to `PKGBUILD` on ArchLinux than making things by yourself, so I'm not going to maintain Debian (also there's no proper packages for dependencies to install it on Debian).

Still not closing an issue, but I flag it now as *wontfix* and I will close it eventually after I test my *generic linux* installation script, unless you have any other questions.

**Comment by MrCoolAndroid** [comment]
@SpacingBat3 Hey! I wasn't able to install it also (from devel branch) but there's no problem, thanks anyways

**Comment by SpacingBat3** [comment]
> @SpacingBat3 Hey! I wasn't able to install it also (from devel branch) but there's no problem, thanks anyways 

@MrCoolAndroid I've tested the script and it were so much broken that I decided to rewrite it again. Also I'm pretty sure Debian `bash` and `sh` != Arch Linux `bash` and `sh`, as on Debian I had syntax errors that were not a case while running same script on Arch Linux.

I'll try to rewrite both of the scripts (`install.sh` and `install-gen`) to make them support Debian.
